pkg/api/annotations.go

```diff
@@ -178,7 +189,15 @@ func UpdateAnnotation(c *models.ReqContext) response.Response {
 		return resp
 	}
 
-	if canSave, err := canSaveByDashboardID(c, annotation.DashboardId); err != nil || !canSave {
+	var canSave bool
```

Comment:
Same as before we need `true` as default
```suggestion
	canSave := true
```